### ISSUE TITLE
Jupiter notebook doesn't show scatter plots inline (pops open in a new window)

### ISSUE THREAD

**Issue by srikris** [apple/turicreate#327]
```
import turicreate
turicreate.show(turicreate.SArray(range(10)), turicreate.SArray(range(10)))
```

**Comment by srikris** [comment]
Same applies to box plots and categorical heatmaps.

```
turicreate.show(turicreate.SArray(range(10), str), turicreate.SArray(range(10), str))
turicreate.show(turicreate.SArray(range(10), int), turicreate.SArray(range(10), str))
```

**Comment by srikris** [comment]
Fixed by #333
